apps/aecore/test/aec_pow_sim.erl

```diff
@@ -0,0 +1,82 @@
+%%%=============================================================================
+%%% @copyright (C) 2017, Aeternity Anstalt
+%%% @doc
+%%%   Simulator for Proof of Work and difficulty adjustments.
+%%% @end
+%%%=============================================================================
+-module(aec_pow_sim).
```

Comment:
Can this file either be called by aec_pow_tests or be hooked to CI somehow?

Reply:
aec_pow_tests already exists? This is just a bit of code we want to keep for future use when investigating algorithms for target adjustment?!